### ISSUE TITLE
[Feature]: Climate card option to show target temp

### ISSUE THREAD

**Issue by msp1974** [piitaya/lovelace-mushroom#636]
### Is your feature request related to a problem?

No

### Describe the solution you'd like

Hi,  firstly amazing job and a huge effort to make such an amazing set of cards!  I use and help write the Drayton Wiser Heating Integration, so understand the time you must have put into this.

On the climate card, would it be possible to have an option to show target temp instead of the current temp, so you can see what the heating is set to.  We find on the Wiser smart TRVs that current temp is pretty inaccurate due to their proximity to the radiator/heat source.

Thanks
Mark

**Comment by mindgam3s** [comment]
+1

**Comment by bennyb0i** [comment]
Agreed, it's a bit awkward only being able to show the current temperature and not being able to instead display the temperature setpoint. Would be great to see the temperature setpoint as an option at least.
